pkg/pillar/scripts/device-steps.sh

```diff
@@ -328,9 +215,13 @@ echo "$(date -Ins -u) Starting nodeagent"
 $BINDIR/nodeagent -c $CURPART &
 wait_for_touch nodeagent
 
-# Restart watchdog - just for ledmanager so far
-killwait_watchdog
-/usr/sbin/watchdog -c $ZTMPDIR/watchdogled.conf -F -s &
+mkdir -p "$WATCHDOG_PID" "$WATCHDOG_PID"
```

Comment:
Why are we creating the same directory twice? It doesn't hurt, since it does `mkdir -p`, but it is confusing.

Reply:
Did you mean this to be this?

```sh
mkdir -p "$WATCHDOG_PID" "$WATCHDOG_FILE"
```

Reply:
Great catch @deitch -- fixed